_articles/ca-ES/best-practices.md

```diff
@@ -1,283 +1,283 @@
 ---
-locale: en-US
-title: Best Practices for Maintainers
-description: Making your life easier as an open source maintainer, from documenting processes to leveraging your community.
-class: best-practices
+locale: ca-ES
+title: Millors pràctiques per a mantenidors
```

Comment:
Debat; fem servir el femení tal com ha fet l'@ygneo? Tu que opinas @ygneo ? i la resta? @enricostano @sseerrggii

Reply:
He arribat a la conclusió de que té sentit usar plural i singular femení nel sentit de "personas" i "persona". Tinc las mevas dubtes del ús gramatical de aixó, pero cap dubte del valor polític que té. Per aquest tipus de documents, crec que el valor politic supera al gramatical. :)

Reply:
Totalment dacord sobre el valor polític, però no sona extrany dir "mantenidores"? A què ens referim, a "persones mantenidores" o a "dones mantenidores"?

Reply:
Entiendo lo que dices. En castellano pasa lo mismo. Chirría gramaticalmente. 

Creo que depende de a qué quieras darle más importancia. Yo he decidido dársela al aspecto político sobre el gramatical. Pienso que cuando has leído un poco de texto y tienes contexto, suele desaparecer la disonancia.

De todos modos si queréis se puede añadir un "disclaimer" explicando por qué se usa la forma femenina. O usar el plural masculino, que al fin y al cabo coincide con el neutro en la gramática actual, al menos en la castellana, imagino que en la catalana también.

Com volgueu. :)